CONTRIBUTING.md

```diff
@@ -16,15 +16,15 @@ If possible, please also add Acceptance Criteria so it is clear what is required
 
 ## Making Changes
 
-The production website, https://metabolicatlas.org, is based on the `main` branch. For development, new work should be based on the `develop` branch.
+The production website, https://metabolicatlas.org, is based on the `main` branch. All new development should be based on this branch.
```

Comment:
Are we going to have a `release` branch? If true, the production web site might be better based on the `release` branch, if not, shouldn't it be based on the latest release tag?

Reply:
I think we should discuss this during the next standup!